src/main/js/page_objects/blueocean/bluePipelineRunDetail.js

```diff
@@ -224,8 +224,11 @@ module.exports.commands = [{
         self.waitForElementVisible('@tests');
         return self;
     },
-    clickFirstResultItem: function (expand = true) {
+    clickFirstResultItem: function (expand) {
         var self = this;
+        if (!expand) {
+            expand = true;
+        }
```

Comment:
Doesn't this code mean that it will be impossible to set expand to false? Perhaps the check should be `expand === undefined` ?